### PR TITLE
Request for freeRTOS taskcreation branch

### PR DESCRIPTION
**Pull request by priyankakumari22k**
freeRTOS tasks with different cases and implemented different freeRTOS API functions.

### CODE REVIEW

**Review by akashk-eic — APPROVED** [review]
Looks good to me.

**Comment by akashk-eic** [comment]
@kriya22k  Please make the pull request header meaningful one.

**Comment by priyankakumari22k** [comment]
[like]  Priyanka Kumari reacted to your message:
________________________________
From: akashk-eic ***@***.***>
Sent: Monday, June 5, 2023 11:32:27 AM
To: kriya22k/freertos ***@***.***>
Cc: Priyanka Kumari ***@***.***>; Mention ***@***.***>
Subject: [External] Re: [kriya22k/freertos] sample file for just for pull request (PR #1)


CAUTION: This email originated from outside of the organization. This message might not be safe, use caution in opening it. If in doubt, do not open the attachment nor links in the message.


@kriya22k<https://github.com/kriya22k> Please make the pull request header meaningful one.

File: STM32CubeIDE/Debug/Application/User/main.su [hunk 1]
```diff
@@ -1,7 +1,17 @@
+<<<<<<< HEAD
```
Comment (akashk-eic) [inline STM32CubeIDE/Debug/Application/User/main.su]: Why this merge conflict related changes are present?

**Review by akashk-eic — CHANGES_REQUESTED** [review]

**Review by akashk-eic — APPROVED** [review]
Looks good.

**Pull request merged**